src/ucp/core/ucp_worker.c

```diff
@@ -456,11 +456,6 @@ static unsigned ucp_worker_iface_err_handle_progress(void *arg)
 
     ucs_assert(ucp_ep->flags & UCP_EP_FLAG_FAILED);
 
-    /* the EP can be closed from last completion callback */
-    ucp_ep_discard_lanes(ucp_ep, status);
-    ucp_ep_reqs_purge(ucp_ep, status);
-    ucp_stream_ep_cleanup(ucp_ep);
```

Comment:
didnt we decide to call use err handler directly from error handler as well? why it was not possible?